backend/src/tools/api.py

```
@@ -0,0 +1,617 @@
+        queryParams = None
+        if operation.get('requestBody'):
+            ref = operation.get('requestBody').get('content').get('application/json').get('schema').get('$ref')
+            if ref:
+                schema = resolve_ref(spec, ref)
+                fully_resolved_schema = resolve_ref_recursive(spec, schema)
+                reqBody = create_schema(fully_resolved_schema.get('title'), fully_resolved_schema.get('properties'))
+                
+            else:
+                model_title = operation.get('operationId').replace("_", " ").replace("-", " ").title().replace(" ", "") + "Args"
+                schema = operation.get('requestBody').get('content').get('application/json').get('schema')
+        model = create_schema(model_title or "DynamicArgs", schema)
```

Pylint (3.3.7)

[error] Using variable 'model_title' before assignment

(E0601)